### ISSUE TITLE
Jaeger exporter marking traces with unknown sampling type

### ISSUE THREAD

**Issue by avishefi** [open-telemetry/opentelemetry-collector-contrib#14124]
**Describe the bug**
The OpenTelemetry Collector demo sends OTLP to the collector and from there an exporter is defined to send traces to Jaeger. Spans and traces reach to Jaeger based on debug logs and Jaeger metrics which keep increasing. However, the traces have a sampling type "unknown" in Jaeger which do not appear in the Jaeger query UI.

**What did you expect to see?**
Traces appear in Jaeger UI.

**What did you see instead?**
Traces reach Jaeger collector, but are marked with sampling_type="unknown" in Jaeger Prometheus metrics.

**What version did you use?**
OpenTelemetry collector 0.59.0
Jaeger 1.37

**What config did you use?**
```yaml
extensions:
  health_check:

receivers:
  otlp:
    protocols:
      grpc:

processors:
  batch:

exporters:
  jaeger:
    endpoint: jaeger:14250
    tls:
      insecure: true

  prometheus:
    endpoint: "0.0.0.0:18889"

  logging:
    loglevel: debug

service:
  extensions: [health_check]

  pipelines:
    traces:
      receivers: [otlp]
      processors: [batch]
      exporters: [jaeger, logging]

    metrics:
      receivers: [otlp]
      processors: [batch]
      exporters: [prometheus]

  telemetry:
    logs:
      level: "debug"
    metrics:
      address: ":6668"
```

**Environment**
OS: CentOS Stream 8

**Comment by jpkrohling** [comment]
@frzifus, can you take this one?

**Comment by frzifus** [comment]
Sure, but it might takes 1-2 days.

**Comment by frzifus** [comment]
hi @avishefi, just to make sure, did you try this[0] demo? 

0: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/v0.59.0/examples/demo

**Comment by avishefi** [comment]
@frzifus Yes, I'm using this demo client and server, but I have removed the all-in-one Jaeger in the docker compose and modified the configuration so that it exports jaeger traces to a production Jaeger instance version 1.37 with Elasticsearch.

**Comment by fatsheep9146** [comment]
@frzifus hi, any update on this issue ?

**Comment by avishefi** [comment]
This was fixed after upgrading to Jaeger 1.38.1 and otel collector 0.62.0.
